model-engine/model_engine_server/infra/gateways/celery_task_queue_gateway.py

```diff
@@ -68,12 +70,16 @@ def send_task(
     ) -> CreateAsyncTaskV1Response:
         celery_dest = self._get_celery_dest()
 
-        res = celery_dest.send_task(
-            name=task_name,
-            args=args,
-            kwargs=kwargs,
-            queue=queue_name,
-        )
+        try:
+            res = celery_dest.send_task(
+                name=task_name,
+                args=args,
+                kwargs=kwargs,
+                queue=queue_name,
+            )
+        except botocore.exceptions.ClientError as e:
```

Comment:
Is there a more specific error we can catch so we don't inadvertently classify internal bugs as 400s?

Reply:
yes i can check the error message whether it's about message size. but for now i want to make it simple and count every boto client error as user errors :) we can revisit it later when users complain.